src/intl/en.json

```diff
@@ -7,6 +7,24 @@
   "cookie-policy": "Cookie Policy",
   "ethereum": "Ethereum",
   "ethereum-logo": "Ethereum logo",
+  "logo": "logo",
+  "uniswap-logo": "Uniswap logo",
```

Comment:
I feel like these `"[project_xyz] logo"` translations are unnecessary. Similar to how we don't translate most proper nouns (e.g. people/project/product names). Seems this creates a bunch of redundant strings for our translators.

Would not translating `"logo"` be sufficient here?

Reply:
I've laid out my argument against this in #2049 – if we don't want to bias against non-English speaker we should translate these as proper strings. Even thought they contain a proper noun the fact that the string includes "logo" makes it a sentence with a possessive. This will translate incorrectly if we fragment the string. I don't believe doing it this way will cause more than a few mins more work for translators, max.